rqd/rqd/cuerqd.py

```diff
@@ -135,137 +108,134 @@ def rebootNow(self):
+    parser.add_argument('--getproxy', metavar='frameID', nargs='+', help='Returns the proxy for the given frameid (debug)')
+    parser.add_argument('--test_edu_frame',action='store_true', help='Launch a edu frame test on an idle core (or first core if none are available)')
+    parser.add_argument('--test_script_frame', action='store_true', help='Launch a script frame test on an idle core (or first core if none are available)')
+    parser.add_argument('--test_script_frame_mac', action='store_true', help='Launch a script frame test for mac on an idle core (or first core if none are available)')
+    
+    args = parser.parse_args()
+
+    rqdHost = RqdHost(args.host)
+     
+    if args.s:
+        print(rqdHost.status())
+    if args.v:
+        tagPrefix = 'rqdv-'
+        for tag in rqdHost.status().host.tags:
+            if tag.startswith(tagPrefix):
+                print("version =", tag[len(tagPrefix):])
+    if args.nimbyoff:
+        rqdHost.nimbyOff()
+    if args.nimbyon:
+        rqdHost.nimbyOn()
+    if args.lp is not None:
+        for arg in args.lp:
+            rqdHost.lock(arg)
+    if args.ulp is not None:
+         for arg in args.ulp:
+            rqdHost.unlock(arg)
+    if args.lh is not None:
+        rqdHost.lockAll()
+    if args.ulh is not None:
+        rqdHost.unlockAll()
+    if args.exit_now:
+        rqdHost.shutdownRqdNow()
+    elif args.exit:
+        rqdHost.shutdownRqdIdle()
+    if args.restart_now:
+        rqdHost.restartRqdNow()
+    elif args.restart:
+        rqdHost.restartRqdIdle()
+    if args.reboot_now:
+        rqdHost.rebootNow()
+    elif args.reboot:
+        rqdHost.rebootIdle()
+    if args.kill is not None:
+        for arg in args.kill:
+            rqdHost.killFrame(arg, "Killed by %s using cuerqd.py" % os.environ.get("USER"))
+    if args.getproxy is not None:
+        for arg in args.getproxy:
+            frameProxy = rqdHost.getRunningFrame(arg)
+            print(frameProxy)               
+    if args.test_edu_frame:
+        print("Launching edu test frame (logs to /mcp)")
+        frameNum = "0001"
+        runFrame = rqd.compiled_proto.rqd_pb2.RunFrame()
+        runFrame.job_id = "SD6F3S72DJ26236KFS"
+        runFrame.job_name = "edu-trn_jwelborn-jwelborn_teapot_bty"
+        runFrame.frame_id = "FD1S3I154O646UGSNN%s" % frameNum
+        runFrame.frame_name = "%s-teapot_bty_3D" % frameNum
+        runFrame.command = "/usr/bin/env VNP_APPLICATION_TIME=1197683283873 /usr/bin/env VNP_VCR_SESSION=3411896 /usr/bin/env PROFILE=default /shots/edu/home/perl/etc/qwrap.cuerun /shots/edu/trn_jwelborn/cue/jwelborn olrun /shots/edu/trn_jwelborn/cue/cue_archive/edu-trn_jwelborn-jwelborn_teapot_bty/v4/teapot_bty.outline %d -batch -event teapot_bty_3D" % int(frameNum)
+        runFrame.user_name = "jwelborn"
+        runFrame.log_dir = "/mcp" # This would be on the shottree
```

Comment:
@bcipriano I was under the impression that /mcp was no longer in use and we'd replaced it with something else?

Reply:
We're transitioning away from the naming, but this is just some test code so its fine as a placeholder value IMO.

We should either updating these testing flags to be less SPI-specific for just remove them if they aren't used anymore. But that's out of scope of this PR.

Reply:
Thanks for the context.